lib/ex_ami/message.ex

```diff
@@ -18,12 +18,7 @@ defmodule ExAmi.Message do
   def new_message(attributes, variables), do: Message.new(attributes, variables)
 
   def new_action(name) do
-    action_id =
-      :os.timestamp()
-      |> Tuple.to_list()
-      |> Enum.map(&Integer.to_string(&1))
-      |> Enum.reduce("", &(&2 <> &1))
-
+    action_id = to_string(:erlang.monotonic_time()) <> to_string(:rand.uniform(1000))
```

Comment:
Integers can be used with string interpolation:
```suggestion
    action_id = "#{:erlang.monotonic_time()}#{:rand.uniform(1000)}"
```

Reply:
both are valid. current solution ok